src/bin/pgcopydb/ld_stream.c

```diff
@@ -91,7 +91,29 @@ stream_init_specs(StreamSpecs *specs,
 
 		case STREAM_PLUGIN_WAL2JSON:
 		{
-			KeyVal options = {
+			KeyVal optionsNumericAsString = {
+				.count = 7,
+				.keywords = {
+					"format-version",
+					"include-xids",
+					"include-schemas",
+					"include-transaction",
+					"include-types",
+					"numeric-data-types-as-string",
+					"filter-tables"
+				},
+				.values = {
+					"2",
+					"true",
+					"true",
+					"true",
+					"true",
+					"true",
+					"pgcopydb.*"
+				}
+			};
+
+			KeyVal optionsNumericAsNumber = {
```

Comment:
Is there a way to share that block of code? Usually I would use a `#define` and refer to it in both arrays, like it's done in other places in the code (have a look at `srcSettings` definition in `copydb.c` for instance.

Reply:
I see your point. I actually took a look at `srcSettings` before writing this piece of code, and I gave up after writing some even uglier piece of code.

I now have an alternative implementation where I have a single struct that I update by appending one option and increment the counter if a condition holds. I guess this is as clean as it can get.

If I were to use `#if` or similar directive on 3 lines, I'd have 6 lines in total. Now I have 7 non-empty lines and 2 of those only contain curly brackets and spaces. I guess this is the cleanest it can get.